recipes/uncommitted/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "uncommitted" %}
+{% set version = "1.6" %}
+{% set md5 = "87a357627ed16a9f02498cea9c61cbb0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [win and py3k]
+  preserve_egg_dir: True
+  entry_points:
+    - uncommitted=uncommitted.command:main
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - uncommitted
+
+  commands:
+    - uncommitted --help
+
+about:
+  home: https://github.com/brandon-rhodes/uncommitted
+  license: MIT
+  license_family: MIT
+  # license_file: LICENSE # When added to MANIFEST.in – https://github.com/brandon-rhodes/uncommitted/pull/13
+  summary: 'Scan filesystem for changes not committed to version control'
+  dev_url: https://github.com/brandon-rhodes/uncommitted
+
+extra:
+  recipe-maintainers:
+    - proinsias
```

Comment:
Good afternoon! I have never maintained a conda recipe before. (And have not figured out why conda duplicates information for which the `setup.py` seems like it should be the system of record?) Given how far behind I am on the pull requests and issues for the GitHub projects I have already, I should probably not take on any additional responsibilities — just keeping the Uncommitted `setup.py` up to date is enough of a challenge! Thanks for letting me know this recipe exists, though.